src/components/container/DailyDataChart/DailyDataChart.tsx

```diff
@@ -241,12 +266,26 @@ export default function DailyDataChart(props: DailyDataChartProps) {
                     <BarChart width={400} height={400} data={data} syncId="DailyDataChart" >
                         <defs>
                             <linearGradient id={gradientKey} x1="0" y1="0" x2="0" y2="1">
-                                <stop offset="0%" stopColor={(props.options as BarChartOptions)?.barColor || "var(--mdhui-color-primary)"} stopOpacity={1.0} />
-                                <stop offset="100%" stopColor={(props.options as BarChartOptions)?.barColor || "var(--mdhui-color-primary)"} stopOpacity={0.7} />
+                                <stop offset="0%" stopColor={resolveColor(layoutContext.colorScheme, (props.options as BarChartOptions)?.barColor) || "var(--mdhui-color-primary)"} stopOpacity={1.0} />
+                                <stop offset="100%" stopColor={resolveColor(layoutContext.colorScheme, (props.options as BarChartOptions)?.barColor) || "var(--mdhui-color-primary)"} stopOpacity={0.7} />
                             </linearGradient>
+
+                            {(props.options as BarChartOptions)?.thresholds?.map((threshold, index) =>
+                                <linearGradient id={gradientKey + "_threshold" + index} x1="0" y1="0" x2="0" y2="1">
+                                    <stop offset="0%" stopColor={resolveColor(layoutContext.colorScheme, threshold.overThresholdBarColor) || "var(--mdhui-color-warning)"} stopOpacity={1.0} />
+                                    <stop offset="100%" stopColor={resolveColor(layoutContext.colorScheme, threshold.overThresholdBarColor) || "var(--mdhui-color-warning)"} stopOpacity={0.7} />
+                                </linearGradient>
+                            )}
                         </defs>
+                        {(props.options as BarChartOptions)?.thresholds?.filter(t=>t.referenceLineColor)?.map((threshold, index) =>
+                            <ReferenceLine y={threshold.value} stroke={resolveColor(layoutContext.colorScheme, threshold.referenceLineColor)} />
```

Comment:
Should a default color be specified for the referenceLine also?

Reply:
If the color isn't specified, the reference line doesn't appear actually